examples/pytorch/text-generator/README.md

```diff
@@ -126,19 +111,26 @@ Cortex can automatically provision infrastructure on your AWS account and deploy
 $ cortex cluster up
 ```
 
-This creates a Cortex cluster in your AWS account, which will take approximately 15 minutes.
+This creates a Cortex cluster in your AWS account, which will take approximately 15 minutes. After your cluster is created, you can deploy to your cluster by using the same code and configuration as before:
 
-After your cluster is created, you can deploy your model to your cluster by using the same code and configuration as before:
+```python
+import cortex
 
-```bash
-$ cortex deploy --env aws
+cx = cortex.client("aws")
```

Comment:
Will this automatically pick up AWS credentials for connecting to the cluster, e.g. from your ~/.aws? Or only from your env vars? And if only from env vars, should we mention that you need to export them, since I think generally people don't export them (or at least I don't, but maybe I am unusual)?